File: HotDesk_SPICE/App_Module/Title.ascx.cs
public partial class App_Module_Title : System.Web.UI.UserControl
{
    private bool _audit = false;
    public bool Audit
    {
        set { _audit = value; }
    }

    protected void Page_Load(object sender, System.EventArgs e)
    {
        if (_audit == false)
        {
            var setting = (Control.Base)this.Page;
            this.lblFormTitle.Text = setting.DisplayTitle + (setting.Action != Control.Base.EnumAction.None ? " - " : string.Empty) + setting.ActionDesc;
        }
        else
        {
            var setting = (Control.LogBase)this.Page;
            this.lblFormTitle.Text = setting.DisplayTitle;
        }
    }
}

File: TMS_HotDesk/App_Module/Title.ascx.vb

Partial Class App_Module_Title
    Inherits System.Web.UI.UserControl

    Private _audit As Boolean = False
    Public WriteOnly Property Audit() As Boolean
        Set(ByVal value As Boolean)
            _audit = value
        End Set
    End Property

    Protected Sub Page_Load(ByVal sender As Object, ByVal e As System.EventArgs) Handles Me.Load
        If _audit = False Then
            Dim setting As Control.Base = CType(Me.Page, Control.Base)
            Me.lblFormTitle.Text = setting.DisplayTitle & If(setting.Action <> Control.Base.EnumAction.None, " - ", String.Empty) & setting.ActionDesc
        Else
            Dim setting As Control.LogBase = CType(Me.Page, Control.LogBase)
            Me.lblFormTitle.Text = setting.DisplayTitle
        End If
    End Sub
End Class
